.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
```

Comment:
What do you think of adding a "dry-run" flag for testing the release, like in [compose-cli](https://github.com/docker/compose-cli/blob/main/.github/workflows/cli-release.yaml#L9)?

Reply:
Fancy! I stole it and made it fancier (✨ types ✨)